app/controllers/vbeta/facilities/va_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'common/models/collection'
+require 'will_paginate/array'
+
+class Vbeta::Facilities::VaController < FacilitiesController
+  before_action :validate_params, only: [:index]
+
+  # Index supports the following query parameters:
+  # @param bbox - Bounding box in form "xmin,ymin,xmax,ymax" in Lat/Long coordinates
+  # @param type - Optional facility type, values = all (default), health, benefits, cemetery
+  # @param services - Optional specialty services filter
+  def index
+    resource = BaseFacility.query(params).paginate
+    render json: resource.data,
+           serializer: CollectionSerializer,
+           each_serializer: VAFacilitySerializer,
+           meta: resource.metadata
+  end
+
+  def show
+    results = BaseFacility.find_facility_by_id(params[:id])
+    raise Common::Exceptions::RecordNotFound, params[:id] if results.nil?
+    render json: results, serializer: VAFacilitySerializer
+  end
+
+  # just for testing/validation, will not stay long term
+  def all
```

Comment:
Lets put a "FIXME" comment in here and link to an issue that we can reference about when/how this action will be removed.